.sbtopts

```diff
@@ -0,0 +1,4 @@
+-J-Xmx2048M
```

Comment:
Not really related to this PR, but :+1: might be nice to include this ;)

Reply:
Oh, this came in by accident. I've added it locally to get akka to compile (we configure sbt memory per project).

I canl remove it from the PR in the next update.